sdk/cognitiveservices/Vision.Face/src/Microsoft.Azure.CognitiveServices.Vision.Face.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Cognitive Services Face SDK</AssemblyTitle>
     <Description>This client library provides access to the Microsoft Cognitive Services Face APIs.</Description>
-    <VersionPrefix>2.4.0</VersionPrefix>
+    <VersionPrefix>2.5.0-preview</VersionPrefix>
```

Comment:
This should only be set to 2.5.0 without the "-preview" as that gets set when producing the build. See https://github.com/Azure/azure-sdk-for-net/blob/master/Documentation/Versioning.md.